index.html

```diff
@@ -1,6 +1,7 @@
 <!doctype html>
 
 <html lang="en">
+
 <head>
   <meta charset="utf-8">
```

Comment:
Don't forget to add your viewport meta tag. 

https://developer.mozilla.org/en-US/docs/Mozilla/Mobile/Viewport_meta_tag